hypervisor/arch/x86/pagetable.c

```diff
@@ -264,7 +264,7 @@ int mmu_modify_or_del(uint64_t *pml4_page,
 	dev_dbg(ACRN_DBG_MMU, "%s, vaddr: 0x%llx, size: 0x%llx\n",
 		__func__, vaddr, size);
 	vaddr_end = vaddr + size;
-	for (; vaddr < vaddr_end; vaddr = vaddr_next) {
+	while (vaddr < vaddr_end) {
```

Comment:
so you mean we cann't use empty operations in “for". Is this the case?
If yes, how about we do:
for (vaddr_end = vaddr + size; vaddr < vaddr_end; vaddr = vaddr_next) {
}
??

Reply:
The static checker requires that the loop incrementation should be simple, like ++ or -- operators.
So, 'vaddr = vaddr_next' is not a good loop incrementation cause.
I discussed this kind of issue with Junjie. We think that changing the for loop to a while loop does not affect the code readability and could help eliminate this kind of violation.